packages/viewer/src/lib/scenes/camera.ts

```diff
@@ -9,14 +9,41 @@ import {
   ClippingPlanes,
   FlyTo,
   FrameCamera,
+  FrameCameraBase,
+  FrameOrthographicCamera,
   FramePerspectiveCamera,
   StandardView,
 } from '../types';
+import { isOrthographicFrameCamera } from '../types/frameCamera';
 import { CameraRenderResult } from './cameraRenderResult';
 import { buildFlyToOperation } from './mapper';
 
 const PI_OVER_360 = 0.008726646259972;
 
+export function cameraToVector(
+  vector: Vector3.Vector3,
+  camera: FrameCamera.FrameCamera
+): Vector3.Vector3 {
+  const position = isOrthographicFrameCamera(camera)
+    ? Vector3.add(camera.lookAt, Vector3.negate(camera.viewVector))
+    : camera.position;
+
+  return Vector3.subtract(position, vector);
+}
+
+export function distanceToVectorAlongViewVec(
+  vector: Vector3.Vector3,
+  camera: FrameCamera.FrameCamera
+): number {
+  const viewVector = isOrthographicFrameCamera(camera)
+    ? camera.viewVector
+    : Vector3.subtract(camera.lookAt, camera.position);
+
+  return (
+    Math.abs(Vector3.dot(viewVector, vector)) / Vector3.magnitude(viewVector)
```

Comment:
Yeah, that's fair. This was copied from [here](https://github.com/Vertexvis/vertex-web-sdk/blob/master/packages/viewer/src/lib/scenes/camera.ts#L148-L154) and just pulled the `Math.abs()` that we were already using, but I think it makes more sense for the `rotateCameraAtPoint(...)` method that uses the `distanceToBoundingBoxCenter()` result to take the absolute value since it's the one that cares. I'll update this

Reply:
you might consider renaming it to `signedDist...` to highlight the fact that negative values are a possibliity.